common/config/rush/pnpm-lock.yaml

```diff
@@ -5005,7 +5007,7 @@ packages:
       fs.realpath: 1.0.0
       inflight: 1.0.6
       inherits: 2.0.4
-      minimatch: 3.1.2
+      minimatch: 3.0.4
```

Comment:
not sure why this is downgraded, but 3.0.4 has a vulnerability issue.

Reply:
ooh. should we manually do a `rush update --full` and check?